dictmysql.py

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/python
 # -*-coding: utf-8 -*-
 
-from __future__ import print_function
```

Comment:
I didn't see any use of print function in this module, hence removed it.

Reply:
I will review the rest of this PR later, but print_function enables print() on Python 2.6. Due to the simplicity of this project, adding this line to add 2.6 support is worth.

Reply:
I know. But I checked print() is not used anywhere in this module.